.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "misc/ext/OpenMesh"]
+	path = misc/openmesh/ext/OpenMesh
+	url = https://www.graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh.git
```

Comment:
should we fork it to make sure it stays around?

Reply:
Myabe. Say it does disappear, its quite a high probability that one of us has it cloned. If we do not fork it it is also easier for other developers to pull later version of OpenMesh. 

I am fine with either

Reply:
It is quite annoying if that server goes down... question is how stable it is...
Can talk on monday...

Reply:
Can one have a fork/mirror that automatically syncs to the original (the submodule would still point to a specific commit)?